code/style.css

```diff
@@ -32,52 +131,82 @@ h1 {
 
 select {
   display: block;
-  border: none;
+  border: solid 2px transparent;
+  outline: none;
+  border-radius: 10px;
   font-size: 22px;
   font-family: 'Montserrat';
-  color: var(--secondary);
+  color: #333;
   width: 100%;
   margin: 24px 0;
+  padding: 10px;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
```

Comment:
Without the arrow in the dropdown, it might be harder to understand that it's a menu, and that you can interact with it.